exercises/practice/pov/TODO.md

```diff
@@ -0,0 +1,30 @@
+## Tree logic POV exercise 
```

Comment:
This file needs to be removed from the current folder because otherwise it might be included in the download "package" the student gets.

@norbs57 If you think all or some of those thoughts/ideas are important to keep, please create an issue (e.g. "POV: Ideas for further improvements") and put them in there. Then people that have opinions on this can find it. We will label the issue as "discussion".
In general I trust your judgement on these. If you think the exercise works as you have it now, lets keep it as is and see how it goes. If you would prefer to make some more changes than go ahead.

Reply:
Thanks.  This is my first more major PR and I kept it in to highlight some of the design decisions where I thought there were several viable options.